jabgui/src/main/java/org/jabref/gui/consistency/ConsistencyCheckAction.java

```diff
@@ -1,24 +1,21 @@
 package org.jabref.gui.consistency;
 
-import java.util.List;
 import java.util.Optional;
 import java.util.function.Supplier;
 
-import javafx.concurrent.Task;
-
 import org.jabref.gui.DialogService;
 import org.jabref.gui.LibraryTab;
 import org.jabref.gui.StateManager;
+import static org.jabref.gui.actions.ActionHelper.needsDatabase;
 import org.jabref.gui.actions.SimpleCommand;
 import org.jabref.gui.preferences.GuiPreferences;
 import org.jabref.gui.util.UiTaskExecutor;
 import org.jabref.logic.l10n.Localization;
 import org.jabref.logic.quality.consistency.BibliographyConsistencyCheck;
 import org.jabref.model.database.BibDatabaseContext;
-import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.BibEntryTypesManager;
 
-import static org.jabref.gui.actions.ActionHelper.needsDatabase;
+import javafx.concurrent.Task;
```

Comment:
I assume, you did not follow Step 3 of our setup guide: https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/intellij-13-code-style.html